### PR TITLE
Fix/api reference

### LINKED ISSUE #3508: /debug/pprof and /debug/pprof/all endpoints are undocumented
The release notes for 2.0.5 say: 

> Add /debug/pprof/all HTTP endpoint to gather all profiles at once.

https://docs.influxdata.com/influxdb/v2.1/reference/release-notes/influxdb/#v205-general-availability-2021-04-27

But I don't see usage of this endpoint documented for 2.x

Usage:

```
> curl localhost:8086/debug/pprof/all --output pprof.tar.gz
> tar -xzvf pprof.tar.gz
> ls -l profiles
total 344
-rw-------  1 sarnold  staff  74559 31 Dec  1969 allocs.pb.gz
-rw-------  1 sarnold  staff    908 31 Dec  1969 block.pb.gz
-rw-------  1 sarnold  staff   4640 31 Dec  1969 goroutine.pb.gz
-rw-------  1 sarnold  staff  73867 31 Dec  1969 heap.pb.gz
-rw-------  1 sarnold  staff   2057 31 Dec  1969 mutex.pb.gz
-rw-------  1 sarnold  staff    280 31 Dec  1969 threadcreate.pb.gz
```

### PR DESCRIPTION
**Pull request by jstirnaman**
Closes #3508

    chore: openapi processing

    - remove custom tags and revert to managing tags in influxdata/openapi.
    - remove security scheme content; revert to managing in influxdata/openapi.
    - delete unused decorator
    - fix strip-version-prefix--replace regex
    - revert to bold for h4-h6
    - use docs-release/influxdb-oss release branch for OSS docs.
    - update README

- [ ] Signed the [InfluxData CLA](https://www.influxdata.com/legal/cla/)
  ([if necessary](https://github.com/influxdata/docs-v2/blob/master/CONTRIBUTING.md#sign-the-influxdata-cla))
- [ ] Rebased/mergeable

### CODE REVIEW

**Comment by jstirnaman** [comment]
Will need to update Cloud contract again after https://github.com/influxdata/openapi/pull/317 is approved and merged.

File: api-docs/README.md [hunk 1]
```diff
@@ -26,7 +28,38 @@ api-docs/
...
+ `influxdata/openapi` **master** may contain OSS spec changes not implemented
+ in the current OSS release, we (Docs team) maintain a release branch, `influxdata/openapi`
+**docs-release/influxdb-oss** used to generate OSS reference docs.
+
+To update this branch, do the following:
+
+  1. (Re)base on the commit used by the [latest release of InfluxDB OSS](#how-to-find-the-api-spec-used-by-an-influxdb-oss-version).
+
+  2. Cherry-pick your documentation fixes into the release branch.
+
```
Comment (sunbryely-influxdata) [inline api-docs/README.md:49]: maybe we can add a quick example for cherry picking such as 
```suggestion
# on the release branch
git cherry-pick <your commit hash>
```

File: api-docs/README.md [hunk 2]
```diff
@@ -26,7 +28,46 @@ api-docs/
...
+git checkout docs-release/influxdb-oss
+git cherry-pick <commit hashes>
+git push -f origin docs-release/influxdb-oss
+```
+
+### How to find the API spec used by an InfluxDB OSS version
+
+`influxdata/openapi` does not version the InfluxData API.
+To find the `influxdata/openapi` commit SHA used in a specific version of InfluxDB OSS,
+check that version (commit or tag) of `/scripts/fetch-swagger.sh` in `influxdata/influxdb`,
```
Comment (jstirnaman) [inline api-docs/README.md:64]:
```suggestion
see `/scripts/fetch-swagger.sh` in `influxdata/influxdb`,
```

File: api-docs/README.md [hunk 3]
```diff
@@ -26,7 +28,46 @@ api-docs/
...
+git cherry-pick <commit hashes>
+git push -f origin docs-release/influxdb-oss
+```
+
+### How to find the API spec used by an InfluxDB OSS version
+
+`influxdata/openapi` does not version the InfluxData API.
+To find the `influxdata/openapi` commit SHA used in a specific version of InfluxDB OSS,
+see `/scripts/fetch-swagger.sh` in `influxdata/influxdb`,
+e.g. https://github.com/influxdata/influxdb/blob/v2.2.0/scripts/fetch-swagger.sh#L13=.
```
Comment (jstirnaman) [inline api-docs/README.md:65]:
```suggestion
see `/scripts/fetch-swagger.sh` in `influxdata/influxdb`--for example,
for the `influxdata/openapi` commit used in OSS v2.2.0, see https://github.com/influxdata/influxdb/blob/v2.2.0/scripts/fetch-swagger.sh#L13=.
```

File: api-docs/README.md [hunk 4]
```diff
@@ -58,7 +100,42 @@ cd api-docs
...
+
+| Example                                                                                                | OpenAPI field                                         |                                            |
+|:-------------------------------------------------------------------------------------------------------|-------------------------------------------------------|--------------------------------------------|
+| [Add supplementary documentation](https://docs.influxdata.com/influxdb/cloud/api/#tag/Quick-start)     | `tags: [ { name: 'Quick start', x-traitTag: true } ]` | [Source](https://github.com/influxdata/openapi/master/src/cloud/tags.yml) |
+| [Group and describe related paths](https://docs.influxdata.com/influxdb/cloud/api/#tag/Authorizations) | `tags: [ { name: 'Buckets', description: '...' } ]`   | [Source](https://github.com/influxdata/openapi/master/src/cloud/tags-groups.yml)) |
+
+## How to test your spec or API reference changes
+
 You can use `getswagger.sh` to fetch contracts from any URL.
 For example, if you've made changes to spec files and generated new contracts in your local `openapi` repo, you can use `getswagger.sh` to fetch and process them.
```
Comment (jstirnaman) [inline api-docs/README.md:140]:
```suggestion
For example, if you've made changes to spec files and generated new contracts in your local `openapi` repo, use `getswagger.sh` to fetch and process them.
```

File: api-docs/README.md [hunk 5]
```diff
@@ -58,9 +100,44 @@ cd api-docs
...
+configure the new content YAML file in [`./openapi/content/content.js`](./openapi/content/content.js).
+The content structure and Markdown must be valid OAS.
+
+Then, you'll need to write or update a decorator module for the node and configure the decorator in the plugin,
+e.g. [`./openapi/plugins/docs-plugin.js`](`./openapi/plugins/docs-plugin.js).
+See the [complete list of OAS v3.0 nodes](https://github.com/Redocly/openapi-cli/blob/master/packages/core/src/types/oas3.ts#L529).
+
+`@redocly/openapi-cli` requires that modules use CommonJS `require` syntax for imports.
+
+### How add tag content or describe a group of paths
```
Comment (sanderson) [inline api-docs/README.md:127]:
```suggestion
### How to add tag content or describe a group of paths
```

**Review by sanderson — APPROVED** [review]
Just one typo. Looks good. Thanks for this @jstirnaman

**Pull request merged**